addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -405,11 +405,18 @@ MockServer.include({
                 channel_direct_message: directMessageInfos,
                 channel_private_group: privateGroupInfos,
             },
-            commands: [{
-                channel_types: ["channel", "chat"],
-                help: "List users in the current channel",
-                name: "who",
-            }],
+            commands: [
+                {
+                    channel_types: ["channel"],
+                    help: "List users in the current channel",
+                    name: "who",
+                },
+                {
+                    channel_types: ["livechat"],
+                    help: "See 15 last visited pages",
+                    name: "history",
+                },
+            ],
```

Comment:
There should be no commands on mock server, by default. Tests should define them in `data`.
There is no python model for commands, and the workaround is very ugly...
I wouldn't mind avoid a mock model `mail.channel_command`, which doesn't actually exist in genuine server.

Reply:
Done